ksql-metastore/src/main/java/io/confluent/ksql/metastore/MetaStoreImpl.java

```diff
@@ -112,18 +139,83 @@ public void deleteSource(final String sourceName) {
   @Override
   public void putAll(MetaStore otherMetaStore) {
```

Comment:
I think this is a dead function. Can we get rid of it?

Reply:
Yes, removed it.